.gitignore

```
@@ -13,3 +13,6 @@
 
 # Dependency directories (remove the comment below to include it)
 # vendor/
+
+#build directory
+build/*
```

Ignore the entire directory.
```suggestion
build/
```